src/ansible_compat/_compat.py

```diff
@@ -0,0 +1,31 @@
+"""Internal glue."""
+import logging
+import sys
+from typing import Callable, TypeVar
+
+try:
+    import tenacity
+except ImportError:
+    pass
+
+WrappedFn = TypeVar("WrappedFn", bound=Callable)
```

Comment:
@webknjaz Do you have an idea how to fix the mypy error on Callable. I tried so many approaches, now I need a break.

Reply:
Maybe
```suggestion
WrappedFn = TypeVar("WrappedFn", bound=Callable[Any, Any])
```
?